src/options.cpp

```diff
@@ -1320,6 +1322,31 @@ void show_options(bool ingame)
             g->init_ui();
             if( ingame ) {
                 g->refresh_all();
+
+                DebugLog( D_INFO, DC_ALL ) << "Loaded tileset: " << OPTIONS["TILES"].getValue();
+
+                tile_loading_report(termap, "Terrain", "");
+                tile_loading_report(furnmap, "Furniture", "");
+                tile_loading_report(Item_factory::Item_factory().get_all_itypes(), "Items", "");
```

Comment:
I copied my idea for Item_factory from the existing code using MonsterGenerator. You say it should be empty, but it's not. The code as you commented on here does work.

Knowing that item_controller exists is one of my blind spots being new around here. Thanks for pointing that out!

Filtering out the fake items and nulls is a good idea. I'll leave that for a follow-up, maybe. It looks like plenty of tilesets include tiles for the nulls just in case, so I might leave that info for the tile designers who want to cover all their bases.